engine-tests/src/utils/mod.rs

```diff
@@ -25,17 +37,6 @@ use rlp::RlpStream;
 use std::borrow::Cow;
 use std::sync::Arc;
 
-#[cfg(not(feature = "ext-connector"))]
-use crate::prelude::parameters::InitCallArgs;
-use crate::prelude::parameters::{StartHashchainArgs, SubmitResult, TransactionStatus};
-use crate::prelude::transactions::{
-    eip_1559::{self, SignedTransaction1559, Transaction1559},
-    eip_2930::{self, SignedTransaction2930, Transaction2930},
-    legacy::{LegacyEthSignedTransaction, TransactionLegacy},
-};
-use crate::prelude::{sdk, Address, Wei, H256, U256};
-use crate::utils::solidity::{ContractConstructor, DeployedContract};
```

Comment:
For some reason, the above imports moved in the beginning. I think it was done unintentionally.

Reply:
Looks like autoformat